crates/ethportal-api/src/lib.rs

```diff
@@ -41,7 +41,7 @@ pub use types::{
     },
     discv5::*,
     enr::*,
-    execution::{block_body::*, header::*, receipts::*},
+    execution::{block_body::*, receipts::*},
```

Comment:
I guess this is a breaking change so it'll require a major version bump for the next `ethportal-api` release